routes/index.js

```diff
@@ -168,8 +168,22 @@ router.get('/reportdb/reports', db.getAllReports);
  *         description: A report with the given report_id
  *         schema:
  *           $ref: '#/definitions/report'
+ *   delete:
+ *     tags:
+ *       - report
+ *     description: Deletes a single report if there are no associated report records
+ *     parameters:
+ *       - name: report_id
+ *         description: report id
+ *         in: path
+ *         required: true
+ *         type: integer
+ *     responses:
+ *       204:
+ *         description: The report was successfully deleted
+ *       404:
+ *         description: The report was not found
```

Comment:
I'm guessing `ValidationError` results in 400 status? That happens if the `id` parameter is missing or if there are associated report records

Reply:
Thanks, I've updated the swagger to include the 400 response.